app/views/idv/sessions/new.html.slim

```diff
@@ -35,7 +35,7 @@ h1.h3.my0 = t('idv.titles.session.basic')
     .mt0.mb3.pb-tiny.border-bottom.border-teal
       | #{t('profile.index.ssn')}#{tooltip(t('tooltips.placeholder'))}
     = f.input :ssn, label: t('idv.form.ssn'), required: true, pattern: '^\d{3}-?\d{2}-?\d{4}$',
-      input_html: { class: 'ssn', 'aria-labelledby': 'label-ssn', value: idv_profile_form.ssn },
```

Comment:
there is no `label-ssn` id on the page, so this is an invalid `aria-labelledby` attr